homeassistant/components/thesilentwave/sensor.py

```diff
@@ -0,0 +1,63 @@
+"""Support for TheSilentWave sensors."""
+
+from homeassistant.components.sensor import SensorEntity, SensorStateClass
+from typing import Callable
+
+from .entity import TheSilentWaveEntity
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Set up the sensor platform."""
+    return True
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up the sensor from a config entry."""
+    coordinator = hass.data["thesilentwave"][entry.entry_id]
```

Comment:
Add typing to the functions.

Use `entry.runtime_data` to access the coordinator